libraries/botbuilder-dialogs-adaptive/tests/templates.test.js

```diff
@@ -1,80 +1,80 @@
 const assert = require('assert');
-const {
-    ConversationState,
-    MemoryStorage,
-    TurnContext,
-    TestAdapter,
-    ActivityFactory,
-} = require('botbuilder');
+const { ConversationState, MemoryStorage, TurnContext, TestAdapter, ActivityFactory } = require('botbuilder');
 const { DialogContext, DialogSet } = require('botbuilder-dialogs');
-const { TextTemplate, languageGeneratorKey, TemplateEngineLanguageGenerator, ActivityTemplate, StaticActivityTemplate } = require('../lib');
+const {
+    TextTemplate,
+    languageGeneratorKey,
+    TemplateEngineLanguageGenerator,
+    ActivityTemplate,
+    StaticActivityTemplate,
```

Comment:
I think this is something required. Just let me know and I'll resolve the comment.

Reply:
Same here.
![image](https://user-images.githubusercontent.com/94375175/150193521-abfd8138-3227-49d5-bb3f-08e8cf6ef5d8.png)